### PR TITLE
[stable/rabbitmq] Option to execute 'rabbitmqctl force_boot'

### LINKED ISSUE #13485: [stable/rabbitmq] Recover from "Waiting for Mnesia tables" after all nodes forced shutdown
**Is your feature request related to a problem? Please describe.**
Yes, in our dev/staging environment in AWS we turn-off at night time (set scaling group to 0) the nodes of EKS cluster to save costs and we enabled persistence for rabbitmq, so it was like unexpected shutdown for all rabbitmq nodes and it can't recover with "Waiting for Mnesia tables", i tried setting podManagementPolicy and service.alpha.kubernetes.io/tolerate-unready-endpoints: "true" but it had no effect, still keeps looping with "Waiting for Mnesia tables"

**Describe the solution you'd like**
Can this solution be applied https://github.com/helm/charts/pull/9645#issuecomment-478638566 as an option? We'd really prefer availability over integrity.

**Describe alternatives you've considered**
Don't use persistence.

### PR DESCRIPTION
**Pull request by denis111**
Signed-off-by: Denis Kalgushkin <keygenmusic@gmail.com>

#### What this PR does / why we need it:
See #13485 . Adds an option to execute 'rabbitmqctl force_boot' to force start on unexpectedly shut down cluster.

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #13485

#### Special notes for your reviewer:
Replaces and closes https://github.com/helm/charts/pull/14059

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/rabbitmq/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: rabbitmq
-version: 5.6.0
+version: 5.6.1
```
Comment (juan131) [inline stable/rabbitmq/Chart.yaml]: Please increment the minor version instead: `5.7.0`

**Review by juan131 — CHANGES_REQUESTED** [review]
Thanks for the contribution. Please check my comments

**Comment by denis111** [comment]
Ok, bumped the version, almost conflicting with another version bump :)

**Review by juan131 — APPROVED** [review]
Thanks for addressing the changes. I'm approving your PR

**Pull request merged**